docs/pages/router/reference/url-parameters.mdx

```diff
@@ -1,23 +1,37 @@
 ---
-title: Use search parameters
-description: Learn how to access and modify search parameters in your app.
-sidebar_title: Search parameters
+title: Use URL parameters
+description: Learn how to access and modify route and search parameters in your app.
+sidebar_title: URL parameters
 ---
 
+import { Collapsible } from '~/ui/components/Collapsible';
 import { FileTree } from '~/ui/components/FileTree';
 import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
 
-Search parameters (also known as query params) are serializable fields that can be added to a URL. They are commonly used to pass data between pages. Expo Router provides hooks for accessing and modifying these parameters.
 
-In nested apps, you'll often have **multiple pages mounted** at the same time. For example, a stack has the previous page and current page in memory when a new route is pushed. Because of this, Expo Router provides two different hooks for accessing search parameters:
+URL parameters include both **route parameters** and **search parameters**.  Expo Router provides hooks for accessing and modifying these parameters.
+
+<Collapsible summary={`What is the difference between "route parameters" and "search parameters"?`}>
```

Comment:
I'm not sure I understand why this should be a collapsible. It's an explanation of route and search parameters. It'll be nice to have this as a section of its own with a title like "Difference between route and search parameters".

Reply:
I felt like it was maybe a bit of an aside, since people may be familiar with route vs search parameters already, but I also wanted to answer that potential question for the reader sooner rather than later, so that's what I went with based on some other sections of the docs using it.

I've now added a new header section, but I did put it immediately at the top to once again answer that potential question immediately.  I had to move the "Local versus global URL parameters" header up with this change, but I think it makes good sense to be where it is now.

Reply:
Makes sense! Thanks!